app.js

```diff
@@ -29,6 +29,12 @@ app.use((err, req, res, next) => {
   } else next(err);
 });
 
+app.use((err, req, res, next) => {
+  if (err.code === "42601" || err.custom === "bad query") {
```

Comment:
This is too restrictive. We're just sending any error back on a 400 code which won't always be the case. Please refactor so that the error is assigned a message and a status when the promise is rejected.